### PR TITLE
Create Aiyush-G.md

### PR DESCRIPTION
**Pull request by Aiyush-G**
## Describe your project in 1-3 sentences
The aim of the project is to have three separate 64 x 64 matricies that display messages between each other so the end-user (both my grandparents) who don't have access to mobile phones (which have low-accessibility due to their small screen size) can still communicate with their loved ones easily.
Furthermore, when not being used for sending messages (powered by a flask-server hosting a REST-API ) the device can display artwork, album art (my grandparents love music) & reminders ie. when to take medicine / meal times / carer times etc...

## Hack Club Slack handle
Aiyush Gupta

## Checklist before submitting proposal:

- [x] I am a current high school student (or younger).
- [x] I have submitted an RSVP at <https://hackclub.com/winter>
- [x] I am a member of the Hack Club Slack (Join at <https://hackclub.com/slack>)
- [x] I have self-reviewed my project proposal.

### CODE REVIEW

File: Aiyush-G.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+name: "@Aiyush-G"
+project: "LED Matrix to Connect the Elderly to the Young"
+---
+
+# LED Matrix to Connect the Elderly to the Young
+
+## Summary
+I am planning to build three LED matricies each powered by an Adafruit Matrix Portal which features both an ATSAMD51J19 Cortex M4 processor and an ESP-32 Wifi co-processor allowing me both the computational power needed and wifi connectivitiy for the three devices to communicate between each other. 
+
+
```
Comment (exu3) [inline Aiyush-G.md:11]:
```suggestion
```

File: Aiyush-G.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+The aim of the project is to have three separate 64 x 64 matricies that display messages between each other so the end-user (both my grandparents) who don't have access to mobile phones (which have low-accessibility due to their small screen size) can still communicate with their loved ones easily.
```
Comment (exu3) [inline Aiyush-G.md:12]:
```suggestion
The aim of the project is to have three separate 64 x 64 matrices that display messages between each other so the end-user (both my grandparents) who don't have access to mobile phones (which have low-accessibility due to their small screen size) can still communicate with their loved ones easily.
```

File: Aiyush-G.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+Furthermore, when not being used for sending messages (powered by a flask-server hosting a REST-API ) the device can display artwork, album art (my grandparents love music) & reminders ie. when to take medicine / meal times / carer times etc...
+
+I am really excited to complete this build because despite being a fun project it will have real-world impact and hopefully make my grandparents daily lives more interesting and fun. From a technical perspective, interfacing the microcontroller with the web-server will provide a delightful challenge and allow me to employ best-practices that I've learnt through my other projects.
+
```
Comment (exu3) [inline Aiyush-G.md:16]:
```suggestion
```

File: Aiyush-G.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+| The device should have a range of options for what is displayed on the screen. | To make this project exciting for its' users, there should be a range of options for what can be displayed on screen ie. messaging, art work, football-scores etc...Exciting!                            | Software will be implemented in micro-python since it is both readable and more easy to implement that alternative languages. |
+| The device should be powered 24/7 via mains.                                   | This is important since if the project was running of a LiPo battery it would quickly run out due to large power requirements. Consequently, a RPI USB-C adapter will be used that draws 3A, 5V and 15W. | Any USB-C wall adapter should work but the RPI one has been selected for most ease.                                           |
```
Comment (exu3) [inline Aiyush-G.md:25]: Nice work on this section!

File: Aiyush-G.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+### Steps
```
Comment (exu3) [inline Aiyush-G.md:27]:
```suggestion
### Steps

```

File: Aiyush-G.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+What steps areyou going to take to build it? What tools are you going to use? What will you do first, second, third, etc.?
+
+6. **Implement the software** 
+This will be written before the 10-days of hacking and then troubleshooted within this time. Refer to the flowchart below on what I hope to implement.
+I've yet to decide whether any external software GUI will be as a webpage or I might create it in the open-source engine Godot.
+![Flowchart showing how software will be implemented](https://i.ibb.co/d08bN02/LED-Matrix-drawio.png)
+
+## Budget
+
+What materials will you need for your project? Where will you get them? How much does it cost? Please include all materials, including components you already own. Make sure to factor in shipping costs and sales tax.
```
Comment (exu3) [inline Aiyush-G.md:70]:
```suggestion
```

**Review by exu3 — APPROVED** [review]
Nice work @Aiyush-G! This is a very creative project; I'm excited to see how you build it.

I've left a few minor comments on your proposal, but it's otherwise good to go!

The @hackclub/bank-ops team will follow up with next steps for receiving your grant.

**Review by hackliv — APPROVED** [review]
hey @Aiyush-G this sounds so cool! my grandparents would love something like this. :) I'm dming you now on Slack to chat about next steps for the grant!

**Pull request merged**